docs/readthedocs/requirements.txt

```diff
@@ -1,7 +1,7 @@
-Sphinx==6.1.3
-sphinx-rtd-theme==1.2.0
-sphinxcontrib-plantuml==0.25
-myst-parser==1.0.0
+Sphinx==5.3.0
+sphinx-rtd-theme==1.1.1
+sphinxcontrib-plantuml==0.24
+myst-parser==0.19.1
```

Comment:
Looks like you can keep these on current releases:
```
sphinxcontrib-plantuml==0.25
myst-parser==1.0.0
```
... while downgrading sphinx and sphinx-rtd-theme